src/Writer/Factory/WriterFactory.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
```

Comment:
Thanks @samsonasik You are right. If you do not mind I'd like to suggest that this will be fixed in a separate PR. The docblocks are wrong in every file and I don't want to make this PR too big.

There's also an issue about this: #47